docs/tutorials/ifci_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,95 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+********************************************
+Exporting IFCI Logs Using QGroundControl
```

Comment:
If I understand correctly, this could be generalized to DroneCAN as well. @vertiq-jordan does DroneCAN telemetry end up in the same spot?